include/CL/opencl.hpp

```diff
@@ -3888,7 +3892,8 @@ class SVMAllocator {
      */
     pointer allocate(
         size_type size,
-        typename cl::SVMAllocator<void, SVMTrait>::const_pointer = 0)
+        typename cl::SVMAllocator<void, SVMTrait>::const_pointer = 0,
+        cl_bool map=true)
```

Comment:
Nitpick: this should probably be:

```suggestion
        bool map = true)
```

Alternatively it could be `cl_bool map = CL_TRUE`, but I expect most users will want to pass `false` rather than `CL_FALSE` if they want to suppress the mapping.

Reply:
Done!